docs/learningLab/introToLit/pok.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 9
+---
+
+# Proof of Knowledge
+![image](../../../static/img/ll_pok.png)
+
+Congratulations on completing this Intro to Lit Protocol lesson! Now it's time to put your newly acquired knowledge to the test and showcase your expertise.
+
+[Go to Gateway.xyz to complete this learning assessment!](https://www.mygateway.xyz/credential/1ec37660-5949-44c4-ad52-0a49f4253877)
+
+This quiz serves as a crucial step in solidifying your understanding and demonstrating your competence in this cutting-edge technologies. It encompasses a range of carefully crafted questions that challenge your comprehension of key concepts, principles, and real-world applications of Lit within the web3 ecosystem.
+
+By successfully completing the quiz, you will have the chance to earn a Proof of Knowledge credential, validating your proficiency.
+
+Take the plunge and test your knowledge today! Good luck and may your expertise shine through as you dive deeper into the world of decentralized technologies!
```

Comment:
same comment as above, might suggest changing to 'dive deeper into the Lit Protocol ecosystem' or something more specific to our product